scripts/protoc_gen_plugin.bash

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+# From https://github.com/bufbuild/buf/blob/master/make/go/scripts/protoc_gen_plugin.bash
```

Comment:
Awesome script for generating all grpc bindings in a very flexible and configurable way. Just using in the top level `Makefile` to generate the go bindings for now

Reply:
Uhm, pretty cool. Agree that this recursive proto-gen will be valuable in other repos.